src/deploy/functions/release/fabricator.ts

```diff
@@ -125,6 +125,12 @@ export class Fabricator {
       this.logOpStart("creating", endpoint);
       upserts.push(handle("create", endpoint, () => this.createEndpoint(endpoint, scraper)));
     }
+    for (const endpoint of changes.endpointsToSkip) {
+      upserts.push(handle("skip", endpoint, () => this.skipEndpoint(endpoint)));
+    }
+    if (changes.endpointsToSkip.length) {
+      upserts.push(this.skipEndpointsCompleted(changes.endpointsToSkip));
```

Comment:
This seems a bit roundabout way to log messages.

Do you think we'll need this wrapping around operation in the future. YAGNI would suggest just logging the messages here directly.

Reply:
TBH it did feel / seem roundabout, but didn't want to stray too far from existing conventions. 🤷